### PR TITLE
Improve CMSPluginBase.render documentation

### PR DESCRIPTION
**Pull request by yakky**
I think that suggesting to call the render of the plugin super class should be encouraged and documented

### CODE REVIEW

**Comment by yakky** [comment]
@mkoistinen @czpython @evildmp do you agree on this?

File: docs/how_to/custom_plugins.rst [hunk 1]
```diff
@@ -265,7 +265,7 @@ Now we need to change our plugin definition to use this model, so our new
         cache = False
 
         def render(self, context, instance, placeholder):
-            context['instance'] = instance
+            context = super(HelloPlugin, self).render(context, instance, placeholder)
```
Comment (mkoistinen) [inline docs/how_to/custom_plugins.rst]: Is this necessary/advised? Isn't the context already passed in as a parameter?

Comment (yakky) [inline docs/how_to/custom_plugins.rst]: the context passed as parameter does not have instance / placeholder: they are added in the parent render method

Comment (mkoistinen) [inline docs/how_to/custom_plugins.rst]: OK, thats fine then =)

**Pull request merged**